app/database/group_attempt_store_compute_all_group_attempts.go

```diff
@@ -111,7 +111,8 @@ func (s *GroupAttemptStore) ComputeAllGroupAttempts() (err error) {
 								AS children_non_validated_categories,
 							MAX(aggregated_children_attempts.validated_at) AS max_validated_at,
 							MAX(IF(items_items.category = 'Validation', aggregated_children_attempts.validated_at, NULL))
-								AS max_validated_at_categories
+								AS max_validated_at_categories,
+							IFNULL(AVG(aggregated_children_attempts.score * items_items.weight), 0) AS average_score
```

Comment:
It will not be null actually, but it still will be wrong:
```
mysql> select avg(score) from (select score from groups_attempts union select null) as scores;
+------------+
| avg(score) |
+------------+
|         20 |
+------------+
1 row in set (0.00 sec)

mysql> select score from groups_attempts;                                       +-------+
| score |
+-------+
|    10 |
|    20 |
|    30 |
+-------+
3 rows in set (0.00 sec)
```

Reply:
fixed